samples/attested_tls/common/openssl_utility.cpp

```diff
@@ -0,0 +1,268 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include "openssl_utility.h"
+
+oe_result_t generate_certificate_and_pkey(X509*& certificate, EVP_PKEY*& pkey)
+{
+    oe_result_t result = OE_FAILURE;
+    SSL_CTX_set_ecdh_auto(ctx, 1);
```

Comment:
Will this file be used in non-enclave binary? If not, please remove this function as it's deprecated in the OpenSSL version that we currently support in the enclave. See https://www.openssl.org/docs/manmaster/man3/SSL_CTX_set_ecdh_auto.html#:~:text=SSL_CTX_set_ecdh_auto()%20and%20SSL_set_ecdh_auto()%20are%20deprecated%20and%20have%20no%20effect.